scripts/bootstrap/bootstrap

```diff
@@ -21,13 +21,35 @@ set -eE
 [[ -z $wget_path ]] && wget_path=/home/root/.local/bin/wget
 
 # Path for the systemd unit that mounts Entware to /opt
+[[ -z $old_systemd_mount_path ]] && old_systemd_mount_path=/etc/systemd/system/opt.mount
 [[ -z $systemd_mount_path ]] && systemd_mount_path=/lib/systemd/system/opt.mount
 
 # Path where the actual Entware distribution resides (will be mounted to /opt)
 [[ -z $entware_path ]] && entware_path=/home/root/.entware
 
+# Find out used branch or get default one
+get-branch() {
+    local opkg_conf=$entware_path/etc/opkg.conf
+    # shellcheck disable=SC2002
+    branch=$(cat "$opkg_conf" | grep src/gz | while read -r line; do
```

Comment:
`     branch=$(cat "$opkg_conf" | grep "^src/gz" | while read -r line; do `

The bootstrap script tought i was on stable because of a commented out line saying so. This ignores comments by checking that it **starts** with "src/gz".

Otherwise it is fine.